### PR TITLE
pick-to-branch: Allow fixing trivial conflicts when cherry-picking

### PR DESCRIPTION
**Pull request by t8m**
A trivial enhancement to allow fixing trivial conflicts when cherry-picking.

### CODE REVIEW

File: review-tools/pick-to-branch [hunk 1]
```diff
@@ -112,7 +112,7 @@ git checkout $target
 ORIG_TARGET_HEAD=`git show -s --format="%H"`
 git pull --ff-only `git rev-parse --abbrev-ref  @{u} | sed "s|/| |"`
 CHERRYPICKING=1
-git cherry-pick -e -x $id~$num..$id || (git cherry-pick --abort; exit 1)
+git cherry-pick -e -x $id~$num..$id || (echo 'Fix or Ctrl-d to abort' ; read || exit 1)
```
Comment (paulidale) [inline review-tools/pick-to-branch:115]: This isn't restoring to the pre-cherry pick state, should it?

Comment (t8m) [reply review-tools/pick-to-branch:115]: The cleanup trap function will do it.

Comment (DDvO) [reply review-tools/pick-to-branch:115]: Yeah, the above call of `git cherry-pick --abort` was (meanwhile) redundant.

**Review by DDvO — APPROVED** [review]
LGTM

**Review by paulidale — APPROVED** [review]

**Comment by DDvO** [comment]
This appears ready to merge.

**Comment by t8m** [comment]
Merged. Thank you for the reviews.

**Pull request closed without merging**